scripts/setup-mel-builddir

```diff
@@ -41,6 +41,8 @@ Options:
   -b BUILDDIR  Specify the build directory to create (default: 'build')
   -o MELDIR    Specify the root of the MEL install, where you can find bitbake
                and the layers
+  -a PATHS     Specify colon separated list of additional directories, where
+               you can find layers
```

Comment:
This should indicate that wildcards are valid, in some fashion, and might also want to mention what directories are searched/checked by default -- i.e. "what is this in addition *to*?"